### PR TITLE
Refactor example for button scss

### PR DESCRIPTION
**Pull request by flexbox**
Don't forget to split position `width` & style `color`

### CODE REVIEW

**Comment by tiste** [comment]
Defo!
Thanks for contributing, I'll merge it asap :+1:. My original idea was to give state to a button with a default style (like the one you update). Then adapt it to common css frameworks (such as foundation and bootstrap).

File: src/angular-loading-button.scss [hunk 1]
```diff
@@ -5,57 +5,56 @@
...
+    width: 250px;
+    height: 70px;
+    border: 2px solid #1ECD97;
+    border-radius: 40px;
+    background: transparent;
+    color: #1ECD97;
+    letter-spacing: 1px;
+    font-size: 18px;
+    font-family: 'Montserrat', sans-serif;
+    -webkit-tap-highlight-color: transparent;
```
Comment (kdisneur) [inline src/angular-loading-button.scss:22]: What's about the `-moz` prefix and the no prefix directive ?

Comment (tiste) [inline src/angular-loading-button.scss:22]: Actually, this style comes from a Codrops article, I simply put that lines here to do some tests and focus on development. I'll get back to design when angular directive will be okay.
I'll update it in future commits.

**Comment by tiste** [comment]
@flexbox could you remove trailing white spaces? And if you're on it, could you fix the directives highlighted by @kdisneur?

**Comment by flexbox** [comment]
And you want a coffee with that ?
I think it's a good excercice for you padawan :)

2014-08-08 10:52 GMT+02:00 Baptiste Lecocq notifications@github.com:

> @flexbox https://github.com/flexbox could you remove trailing white
> spaces? And if you're on it, could you fix the directives highlighted by
> @kdisneur https://github.com/kdisneur?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tiste/angular-loading-button/pull/1#issuecomment-51577811
> .

## 

David Leuliette
Front-end Developer http://davidl.fr

**Comment by kdisneur** [comment]
@flexbox black, without sugar please ;)

**Comment by tiste** [comment]
I can't merge a PR with **awful** mistakes such as keeping trailing white spaces, but you taught me many things so, let's merge it :smirk:

**Comment by tiste** [comment]
@flexbox @kdisneur released :wink:

**Pull request merged**